content/cloud-docs/api-docs/index.mdx

```diff
@@ -86,6 +86,7 @@ The following entitlements are available:
 - `cost-estimation` — Allows an organization to access [cost estimation][].
 - `operations` — Allows an organization to perform runs within Terraform Cloud. Affects the [runs][], [plans][], and [applies][] endpoints.
 - `private-module-registry` — Allows an organization to publish and use modules with the [private module registry][]. Affects the [registry modules][] endpoints.
+- `run-tasks` — Allows an organization to use [run task integrations][]. Affects the [run tasks][] endpoints.
```

Comment:
@laurapacilio   This is probably the only "controversial" bit of the PR.  I wasn't quite sure how to distinguish between `run tasks` the integration/feature versus  `run tasks` the API endpoint.

That was the best I could come up with. I also tried `Allows an organization to use Run Task Integrations ....` but I remember in the last PR, features tend not to have uppercase names.  And I can see cost estimates and vcs are both lower case.

Reply:
thank you for flagging this and for caring about capitalization (that makes me happy!). I support the decision to stick with lower case because first, as you mentioned, we do not capitalize feature names. And second, we don't capitalize the things anywhere else here - consistency is good. My only question is actually - I see you're linking to the integrations page, but would it be more helpful  to link here: https://www.terraform.io/cloud-docs/workspaces/settings/run-tasks

The reason why is that the integrations page is really (at least to me) about how to build a run task integration for TFC. But I think we may want the more end user context? Lmk if you completely disagree though!

Reply:
Agreed. That's a better link!

Reply:
Updated